Common subdirectories: acme-0.7.0-orig/acme and acme-0.7.0/acme
Common subdirectories: acme-0.7.0-orig/acme.egg-info and acme-0.7.0/acme.egg-info
Common subdirectories: acme-0.7.0-orig/docs and acme-0.7.0/docs
Common subdirectories: acme-0.7.0-orig/examples and acme-0.7.0/examples
diff -u acme-0.7.0-orig/setup.py acme-0.7.0/setup.py
--- acme-0.7.0-orig/setup.py	2016-05-27 15:19:34.000000000 -0500
+++ acme-0.7.0/setup.py	2016-05-31 09:25:24.237187603 -0500
@@ -18,9 +18,7 @@
     'pyrfc3339',
     'pytz',
     'requests',
-    # For pkg_resources. >=1.0 so pip resolves it to a version cryptography
-    # will tolerate; see #2599:
-    'setuptools>=1.0',
+    'setuptools',
     'six',
 ]
 
